resource/schema/system.sql

```diff
@@ -1,4 +1,4 @@
--- 32
+-- 33
```

Comment:
We actually don't do this anymore, for reasons that I don't recall and don't want to risk relearning. The last system.sql version update was in 2015, and all changes since then have been performed by [schema.js steps with a userdata.sql bump](https://github.com/zotero/zotero/commit/3b402c564cb3bb4a41a9486f8522c8793dfdd2fb).